frontend/src/components/Navigation/Navigation.js

```diff
@@ -1,10 +1,38 @@
 import React from 'react';
-import { Container, Nav } from './Navigation.styled';
+import { FaHome, FaBell, FaUserAlt, FaHeart, FaPlusSquare } from 'react-icons/fa';
+
+import {
+  Container,
+  NavMenu,
+  NavItem,
+  NavLink,
+  NavIcon,
+  NavButton,
+  NavText,
+} from './Navigation.styled';
 
 const Navigation = () => {
+  const navItems = [
+    { path: '/', icon: <FaHome />, title: 'Home' },
+    { path: 'subscribed', icon: <FaHeart />, title: 'Subscribed' },
+    { path: 'addPost', icon: <FaPlusSquare />, title: 'Add post' },
+    { path: 'notifications', icon: <FaBell />, title: 'Notifications' },
+    { path: 'singUp', icon: <FaUserAlt />, title: 'Sing Up!' },
```

Comment:
Sing? Nie, dziękuję, wolę nie śpiewać ;D

Reply:
singUp = zapisać się :D

Reply:
a nie, w oczach mi sie przestawiło, my bad xD

Reply:
SingIn

Reply:
sign in